### PR TITLE
Application aware intentions scenario

### PR DESCRIPTION
**Pull request by danielehc**
Katacoda lab for Application Aware Intentions tutorial.

Demo with same code (up to this PR opening): https://www.katacoda.com/danielehc/scenarios/consul-application-aware-intentions

Learn PR: https://github.com/hashicorp/learn/pull/2449

### CODE REVIEW

File: consul-application-aware-intentions/finish.md [hunk 1]
```diff
@@ -1,4 +1,17 @@
 ## Review
 
-In this hands-on lab, you deployed a Consul datacenter using Docker.
+In this hands-on lab, you operated a Consul datacenter, deployed using Docker, to configure application aware intentions.
```
Comment (DerekStrickland) [inline consul-application-aware-intentions/finish.md:3]:
```suggestion
In this hands-on lab, you created a Consul datacenter, deployed using Docker, and configured application aware intentions.
```

File: consul-application-aware-intentions/intro.md [hunk 2]
```diff
@@ -2,6 +2,13 @@
 
 In this hands-on lab, you will operate a Consul datacenter, deployed using Docker, to configure application aware intentions.
```
Comment (DerekStrickland) [inline consul-application-aware-intentions/intro.md:3]: Is this intro text for both the beginning and finish part actually what you want? It doesn't feel like it goes with the list below.

**Review by DerekStrickland — APPROVED** [review]
Question and suggestion

File: consul-application-aware-intentions/intro.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# Application Aware Intentions With Consul Service Mesh
+
+In this hands-on lab, you will operate a Consul datacenter, deployed using Docker, to configure application aware intentions.
```
Comment (danielehc) [inline consul-application-aware-intentions/intro.md:3]:
```suggestion
In this hands-on lab you will configure and deploy application aware intentions to a pre-existing Consul datacenter.
```

**Pull request merged**